packages/notifications/demo/stories/data.ts

```diff
@@ -13,3 +13,14 @@ export const TOAST_PLACEMENT_OPTIONS = [
   'top-end',
   'top-start'
 ] as const;
+
+export const GLOBAL_ALERT_BUTTONS = [
+  {
```

Comment:
minor question/ask. design's recommendation is to only have one action in the alert (excluding any anchors and close). could we remove this second button? i'm afraid folks might see this and consider this default, and don't want to confuse "what should" with "what's possible"

Reply:
Hey @steue, I appreciate your concern. However, we really need to lean into the purpose of Storybook here – which is to press on the extents of the layout and styling. Indeed, we use Storybook as the testing ground for "what's possible". By contrast, the website is the place to show, by example, "what should" happen with component usage.

Reply:
To put it another way...
- if we remove either button, we run the future risk of missing unintended layout updates that might impact margining
- if we remove the basic button, we run a future risk of missing unintended styling updates to that button
- if we remove the primary button, we run future risk of missing unintended styling updates to that custom button

Reply:
yes that makes sense! definitely want to uphold the purpose of Storybook demos. our website documentation (will be) quite clear on the recommendation, so there shouldn't be any confusion.